yt/visualization/streamlines.py

```diff
@@ -229,7 +229,7 @@ def path(self, streamline_id):
         >>> streamlines = Streamlines(ds, [0.5]*3)
         >>> streamlines.integrate_through_volume()
         >>> stream = streamlines.path(0)
-        >>> matplotlib.pylab.semilogy(stream['t'], stream['Density'], '-x')
```

Comment:
funnily enough, I think this would *not* be ambiguous!

Reply:
I wouldn't, but I went ahead and fixed all occurrences of `Density` altogether, as it may be named something else in other frontends.